snap/README.md

```diff
@@ -534,61 +553,112 @@ root@edgex:~# cd edgex-go && snapcraft --destructive-mode
+For example, the upstream snapcraft docker image can be used (only on x86_64 architectures unfortunately) like so:
 
 ```bash
-$ docker run -it -v"$PWD":/build snapcore/snapcraft:stable bash -c "apt update && cd /build && snapcraft"
+docker run -it -v"$PWD":/build snapcore/snapcraft:stable bash -c "apt update && cd /build && snapcraft"
 ```
 
-Note that if you are building your own docker image, you can't run snapd inside the container, and so to install snapcraft, the docker image must download the snapcraft snap and extract it as if it was installed normally inside `/snap` (same goes for the `core` and `core18` snaps). This is done by the Linux Foundation Jenkins server for the project's CI and you can see an example of that [here](https://github.com/edgexfoundry/ci-management/blob/master/shell/edgexfoundry-snapcraft.sh). The upstream docker image also does this, but only for x86_64 architectures.
+Note that if you are building your own docker image, you can't run snapd inside the container, and so to install snapcraft, 
+the docker image must download the snapcraft snap and extract it as if it was installed normally inside `/snap` 
+(same goes for the `core` and `core18` snaps). 
+This is done by the Linux Foundation Jenkins server for the project's CI and you can see an example of that 
+[here](https://github.com/edgexfoundry/ci-management/blob/master/shell/edgexfoundry-snapcraft.sh). 
+The upstream docker image also does this, but only for x86_64 architectures.
 
 #### Multipass / generic VM
 
-To use multipass to create an Ubuntu 18.04 environment suitable for building the snap (i.e. when running natively on windows):
+To use multipass to create an Ubuntu 20.04 environment suitable for building the snap (i.e. when running natively on windows):
 
 ```bash
-$ multipass launch bionic -n edgex-snap-build
+$ multipass launch focal -n edgex-snap-build
 $ multipass shell edgex-snap-build
 multipass@ubuntu:~$ git clone https://github.com/edgexfoundry/edgex-go
 multipass@ubuntu:~$ cd edgex-go
 multipass~ubuntu:~$ sudo snap install snapcraft --classic
 multipass~ubuntu:~$ snapcraft --destructive-mode
 ```
 
-The process should be similar for other VM's such as kvm, VirtualBox, etc. where you create the VM, clone the git repository, then install snapcraft as a snap and run with `--destructive-mode`. 
+The process should be similar for other VM's such as kvm, VirtualBox, etc. where you create the VM, clone the git repository, 
+then install snapcraft as a snap and run with `--destructive-mode`. 
 
 ### Developing the snap
 
-After building the snap from one of the above methods, you will have a binary snap package called `edgexfoundry_<latest version>_<arch>.snap`, which can be installed locally with the `--devmode` flag:
+After building the snap from one of the above methods, you will have a binary snap package called `edgexfoundry_<latest version>_<arch>.snap`, 
+which can be installed locally with the `--devmode` flag:
```

Comment:
Not introduced by this PR, but since we are changing this section:
dev mode breaks confinement. The flag used here should be `--dangerous`

Reply:
Updated.